lib/cisco_node_utils/cmd_ref/snmpnotification.yaml

```diff
@@ -0,0 +1,11 @@
+# snmpnotification
+---
+enable:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/^(no )?(snmp-server) enable traps <trap_name> ?$/'
+  config_set: "<state> snmp-server enable traps <trap_name>"
+
+notifications:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/.*snmp-server enable traps (.*)?$/'
```

Comment:
i want both enabled and disabled traps. i also need to handle if the trap disappears if disabled. eg bgp

the regex is what it should be